src/game/shared/mapbase/vscript_singletons.cpp

```diff
@@ -103,6 +105,18 @@ extern ISaveRestoreOps* GetStdStringDataOps();
 	#endif
 #endif
 
+#ifndef CLIENT_DLL
+// This is a smaller version of the stock net prop manager which has functions we need
+CNetPropManager g_ScriptNetPropManagerStub;
+#endif
+
+// Copied from netpropmanager.cpp
+const char *s_pszBannedNetProps[]
+{
+	"EntityQuality",
+	"AccountID",
```

Comment:
These should be within `#ifdef TF` so that other games don't unnecessarily check for them.

And I'm wondering, why are these partial lookups? Why not just `m_iEntityQuality`, `m_iAccountID` and `strcmp == 0`?